journal.py

```diff
@@ -201,31 +253,23 @@ def logout(request):
 
 @view_config(route_name='detail', renderer='templates/detail.jinja2')
 def entry_details(request):
-    # call read entries
-    # select out specific entry based on id in uri
-    db_id = request.matchdict.get('id', -1)
-    cursor = request.db.cursor()
-    cursor.execute(SELECT_SINGLE_ENTRY, (db_id,))
-    keys = ('id', 'title', 'text', 'created')
-    entry = dict(zip(keys, cursor.fetchone()))
-    # convert text- markdown into html
-    entry['display_text'] = markdown.markdown(entry['text'], extensions=['codehilite(linenums=True)', 'fenced_code'])
+    entry = Entry.by_id(request.matchdict.get('id', -1))
+    del entry.__dict__['_sa_instance_state']
+    entry.display_text = markdown.markdown(entry.text, extensions=['codehilite(linenums=True)', 'fenced_code'])
     return {'entry': entry, }
 
 
+# ported to ORM, but does not update database
```

Comment:
I feel like this comment is out of date.  You've added the `pyramid_tm` config, that should do the trick.